test/node/connection-event.js

```diff
@@ -42,16 +42,17 @@ test.test('server must process an event', (test) => {
   client.connectAndHandshake(app.name, null, null, (error, connection) => {
     test.assertNot(error, 'must connect to server');
 
-    server.getClients()[0].on('event', (event) => {
-      test.strictEqual(event.interfaceName, iface,
+    server.getClients()[0].on('event',
+      (interfaceName, remoteName, remoteArgs) => {
+        test.strictEqual(interfaceName, iface,
         'event interface must match');
```

Comment:
It will be tracked automatically as soon as we update ESLint to 4.0. The only reason I haven't done it yet is that we use the `eslint-plugin-import` plugin which still has `eslint@3.x` as its peerDependency (though it seems to work with the latest eslint nevertheless).